### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by privatebin-translator**

### CODE REVIEW

File: i18n/nl.json [hunk 1]
```diff
@@ -164,27 +164,27 @@
     "Preparing new paste…": "Nieuwe geplakte tekst voorbereiden…",
     "In case this message never disappears please have a look at <a href=\"%s\">this FAQ for information to troubleshoot</a>.": "In het geval dat dit bericht nooit verdwijnt, kijkt u dan eens naar <a href=\"%s\"> veelgestelde vragen voor informatie over het oplossen van problemen </a>.",
     "+++ no paste text +++": "+++ geen geplakte tekst +++",
-    "Could not get paste data: %s": "Kon geen klembordgegevens verkrijgen: %s",
```
Comment (elrido) [inline i18n/nl.json:167]: this reverts the existing translation

File: i18n/tr.json [hunk 2]
```diff
@@ -1,135 +1,135 @@
 {
     "PrivateBin": "PrivateBin",
-    "%s is a minimalist, open source online pastebin where the server has zero knowledge of pasted data. Data is encrypted/decrypted %sin the browser%s using 256 bits AES.": "%s sunucunun burada paylaştığınız veriyi görmediği, minimal, açık kaynak bir pastebindir. Veriler tarayıcıda 256 bit AES kullanılarak şifrelenir/çözülür.",
```
Comment (elrido) [inline i18n/tr.json:3]: most of the translations here got reverted

**Review by elrido — CHANGES_REQUESTED** [review]
I guess the reverts need to be reverted upstream in Crowdin - Vandalism?

**Comment by r4sas** [comment]
> I guess the reverts need to be reverted upstream in Crowdin - Vandalism?

Looks like someone translated to Ukrainian, but broke other translations.

**Comment by r4sas** [comment]
Okay, I had reverted broken translations... it looks for me like crowdin somehow lost them.

**Comment by r4sas** [comment]
@elrido please check Crowdin for issues reports I've made about approved translations.

**Review by elrido — DISMISSED** [review]

**Comment by elrido** [comment]
> @elrido please check Crowdin for issues reports I've made about approved translations.

Thank you very much! I did some additional minor tweaks (replace `%s` with `%d` and the three dots with the unicode glyph) in some places and added the new Slovak translation that got requested since (I'll still need to do the changes to enable it's use in PB -> in a separate PR). I'll merge what we have so far.

**Pull request merged**